src/DriverTypes.hpp

```diff
@@ -353,16 +353,22 @@ struct DeviceSettings
     // 8096.. 2097152
     std::vector<int> poolSize;
 
-    // Dropped data from transmission buffer (bytes) per channel.
-    // Set 0 for reset counter;
-    // Cases: 1)ResetType; 2)idleTimeout; 3)transmission to remoteAddress 0
-    std::vector<int> dropCount;
+//    // Dropped data from transmission buffer (bytes) per channel.
```

Comment:
If it's not in use, remove the code ! It's in git, you can always get it back later if you need it.